src/GitHubParserHook.php

```diff
@@ -43,14 +53,58 @@ private function setFields( ProcessingResult $result ) {
 		$this->fileName = $params['file']->getValue();
 		$this->repoName = $params['repo']->getValue();
 		$this->branchName = $params['branch']->getValue();
+
+		foreach ( self::$syntaxHighlightFields as $val ) {
+			if ( isset( $params[$val] ) ) {
+				$this->$val = $this->cleanField( $params[$val]->getValue() );
+			}
+			else {
+				$this->$val = null;
+			}
+		}
 	}
 
-	private function getRenderedContent() {
+	private function cleanField( $val ) {
+		if ( $val !== null ) {
+			$val = trim( $val, "'\"" );
+		}
+		return $val;
+	}
+
+	private function getRenderedContent(Parser $parser) {
 		$content = $this->getFileContent();
 
 		if ( $this->isMarkdownFile() ) {
 			$content = $this->renderAsMarkdown( $content );
 		}
+		else if ($this->lang !== "") {
+			if ( ExtensionRegistry::getInstance()->isLoaded( 'SyntaxHighlight' ) ) {
```

Comment:
This was an initial idea to support multiple syntax highlighters, but after looking around most of them aren't  maintained or aren't stable. Of those that are, they use the same config as the one Mediawiki downloads with - I took it out.